core/src/main/groovy/noe/tomcat/configure/SecureHttpConnectorTomcat.groovy

```diff
@@ -44,6 +44,7 @@ public class SecureHttpConnectorTomcat extends ConnectorTomcatAbstract<SecureHtt
 
   // Inner elements
   private ConnectorUpgradeProtocolTomcat upgradeProtocol
+  private ConnectorSSLHostConfigTomcat[] SSLHostConfigs
```

Comment:
Can we specify it as List<ConnectorSSLHostConfigTomcat> please. Because collection access code (I don't see any specific reason to use an array here)

Reply:
That's a good point.